src/options.cpp

```diff
@@ -37,7 +37,7 @@ namespace uncrustify
 namespace options
 {
 
-lineends_e &newlines()
+lineend_e &newlines()
```

Comment:
Don't we use any convention to separate words?

Reply:
Heh, yeah... would you be okay punting on this until after #1852? That will save some rebasing effort. (`tokenpos_e` should be changed also...)

We have a bunch of code floating around that doesn't conform to the proposed naming convention from #1843.